ghost/i18n/locales/pa/newsletter.json

```diff
@@ -0,0 +1,28 @@
+{
+    "{date}": "{date}",
+    "Become a paid member of {site} to get access to all premium content.": "{site} ਦੇ ਸਹਿਯੋਗੀ ਮੈਂਬਰ ਬਣ ਕੇ ਸਾਰੀ ਪ੍ਰੀਮੀਅਮ ਸਮੱਗਰੀ ਤੱਕ ਪਹੁੰਚ ਪ੍ਰਾਪਤ ਕਰੋ।",
+    "By {authors}": "{authors} ਦੁਆਰਾ",
+    "Comment": "ਟਿੱਪਣੀ",
+    "complimentary": "ਮੁਫ਼ਤ",
+    "Email": "ਈਮੇਲ",
+    "free": "ਮੁਫ਼ਤ",
+    "Keep reading": "ਪੜ੍ਹਦੇ ਰਹੋ",
+    "Less like this": "ਇਸ ਵਰਗੇ ਘੱਟ",
+    "Manage subscription": "ਸਦੱਸਤਾ ਦਾ ਪ੍ਰਬੰਧਨ ਕਰੋ",
+    "Member since": "ਮੈਂਬਰ ਜਦੋਂ ਤੋਂ",
+    "More like this": "ਇਸ ਵਰਗੇ ਹੋਰ",
+    "Name": "ਨਾਮ",
+    "paid": "ਭੁਗਤਾਨ ਕੀਤਾ",
+    "Subscription details": "ਸਦੱਸਤਾ ਵੇਰਵੇ",
+    "trialing": "ਅਜ਼ਮਾਇਸ਼ੀ ਸਦੱਸਤਾ",
+    "Unsubscribe": "ਸਦੱਸਤਾ ਰੱਦ ਕਰੋ",
+    "Upgrade": "ਅੱਪਗ੍ਰੇਡ ਕਰੋ",
+    "Upgrade to continue reading.": "ਪੜ੍ਹਨਾ ਜਾਰੀ ਰੱਖਣ ਲਈ ਅੱਪਗ੍ਰੇਡ ਕਰੋ।",
+    "View in browser": "ਬ੍ਰਾਊਜ਼ਰ ਵਿੱਚ ਦੇਖੋ",
+    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "ਤੁਹਾਨੂੰ ਇਹ ਇਸ ਲਈ ਮਿਲ ਰਿਹਾ ਹੈ ਕਿਉਂਕਿ ਤੁਸੀਂ {site} ਦੇ ਇੱਕ <strong>%%{status}%% ਗਾਹਕ</strong> ਹੋ।",
```

Comment:
The terms for complimentary, free, paid, and trialing will be substituted in for status.  AI says that your strings for 'paid' and trialing are not grammatical in this context.

Reply:
AI suggests that ਮੈਂਬਰ might be better than ਗਾਹਕ , since ਮੈਂਬਰ  means more 'customer' than 'subscriber'.  I'm happy to defer to you as the native speaker, but please have a quick look and make sure you're right and not the AI.  😄